### ISSUE TITLE
AI Tools Categorized

### ISSUE THREAD

**Issue by shmbajaj** [shmbajaj/hindustaantech#13]

**Comment by sthakur0501** [comment]
> 100 AI TOOLS

1. Tensorflow

One of Tensorflow’s most appealing features is its ability to support both central processing unit (CPU) and graphics processing unit (GPU) execution. As a result, maintaining GPU performance does not necessitate writing code in C++ or CUDA.

It employs a network of interconnected nodes to speed up the creation, training, and distribution of artificial neural networks capable of handling large datasets. This is used by Google’s image- and speech-recognition applications, respectively.

2. Scikit-learn

Scikit-learn is a program that assists users who are unfamiliar with a well-known but advanced programming language. Its main advantage is that it is distributed under the BSD license and has few dependencies. It promotes Scikit’s use in both professional and academic settings. Another advantage is the tool’s user-friendly interface. All Scikit Learn algorithms use the same function calls to build, test, and use models.

3. Theano

Theano is a well-known open-source Python deep learning toolkit in neural processing and data science. It is well-known for simplifying the creation of sophisticated neural networks by abstracting away the neural network components. It is commonly used on graphics processing units (GPUs) to create and train AI models, and Facebook has used it for both the training and deployment of AI applications.

4. Caffe

AI developed at Berkeley Research Caffe is a deep learning framework created in collaboration with community members. Expression, quickness, and modularity are highlighted in its architecture. It has a strong architecture that allows for configuration-defined models and optimization without the need for manual coding. It can also switch between CPU and GPU.

Caffe is well suited for research projects and industrial deployments because a single NVIDIA K40 GPU can process over 60 million images per day.

5. Keras

Keras, a Python-based neural network library similar to TensorFlow and CNTK, is one of the best AI frameworks, though it is not intended to be an end-to-end machine learning framework. Keras was designed as an application programming interface (API) for humans, not computers.

6. MxNet

The ability of this feature to trade compute time for memory via “forgetful backdrop” is extremely useful for recurrent neural networks training on extremely long sequences. Designed with growth in mind. There are numerous useful options available, such as the ability to quickly create custom layers using high-level programming languages.

7. Pytorch

PyTorch is a Python library that aids in the development of machine learning algorithms. Its primary audience is academic researchers, but developers working with Tensorflow may find it useful.

Medium is a versatile Python framework that can be used to create systems of any size. Because of its complete interface to hardware accelerators and user-friendly application programming interface, the designers claim their framework is the “most intuitive” for system development (API). However, it is notoriously slow in terms of graphics processing units (GPUs).

8. OpenNN

OpenNN offers a variety of sophisticated analytics, ranging from those suitable for a complete beginner setup to those designed for more experienced programmers. Neural Designer, a sophisticated analytics tool, is included. This tool provides graphs and tables to help you understand the data you’ve entered

9. Google AutoML

It is a Google product known as AutoML. Google formally appropriated it in May 2017 for use in their search for neural network architectures. AutoML is one of the most robust and adaptable AI frameworks available. Auto ML provides a simple graphical user interface to assist developers in assessing, improving, training, and releasing models based on their data. They can have their personalized machine-learning models in a matter of minutes. Using Auto ML and Google’s neural architecture search technologies and transfer learning, developers can create custom models for their businesses.

**Comment by shmbajaj** [comment]
| Tool Name | Tool Link | Tool Type | Tool Use Case | Tool Examples | Tool Description |
|-----------|-----------|-----------|---------------|---------------|------------------|
| Chat GPT  | [Chat GPT](https://chat.openai.com)      | Chat and Q&A | Ask questions and get answers | [AI Breakfast](https://twitter.com/AiBreakfast) | Chat GPT is a computer program that can talk to people like a human and is used to create chatbots and virtual assistants for businesses and developers. |

**Comment by sthakur0501** [comment]
| Tool Name | Tool Website | Tool Type | Tool Use Case | Tool Examples | Tool Description |
Ai chart
